docker-compose.yaml

```diff
@@ -18,6 +18,8 @@ services:
     environment:
       - POSTGRES_USER=${POSTGRES_DB_USER:-postgres}
       - POSTGRES_PASSWORD=${POSTGRES_DB_PASSWORD:-postgres}
+      - POSTGRES_USERNAME=${POSTGRES_DB_USER:-postgres}
+      - POSTGRES_DB=${POSTGRES_DB:-postgres}
```

Comment:
I'd maybe change this to `ffc_mpdp_backend` so it's separate from the default `postgres` database and consistent with higher environments.